Core/Util/GameManager.cpp

```diff
@@ -293,6 +370,7 @@ bool GameManager::InstallGame(std::string zipfile, bool deleteAfter) {
 	installProgress_ = 0.0f;
 	installInProgress_ = false;
 	installError_ = sy->T("Storage full");
+	// Should we really delete in this case???
```

Comment:
I agree - if anything we should delete the files extracted so far....

-[Unknown]

Reply:
Which we do right below here, I think... I'm removing the deletion of the original file.